Training_25/Program.cs

```diff
@@ -3,11 +3,26 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T06 branch. Assignment Q6: Digital Root.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
 namespace Training_25;
+
 internal class Program {
    static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+      WriteLine ("Enter a number, to find it's digital root:");
+      long num;
+      while (!long.TryParse (ReadLine (), out num))
+         WriteLine ("Invalid input.Enter a valid number.");
+      while (num >= 10) {
+         long sum = 0;
+         while (num > 0) {
+            sum += num % 10;
+            num /= 10;
+         }
+         num = sum;
+      }
+      WriteLine ($"Digital root:{num}");

```

Comment:
```suggestion
      WriteLine ($"Digital root: {num}");
```
Please add a space after the colon